cmake/RocksDBConfig.cmake.in

```diff
@@ -1,3 +1,56 @@
 @PACKAGE_INIT@
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/modules")
+
+include(CMakeFindDependencyMacro)
+
+set(GFLAGS_USE_TARGET_NAMESPACE @GFLAGS_USE_TARGET_NAMESPACE@)
+
+if(@WITH_JEMALLOC@)
+  find_dependency(JeMalloc)
```

Comment:
```suggestion
  find_dependency(JeMalloc REQUIRED)
```

Reply:
`find_dependency()` should have that already if user askes for `find_package(RocksDB REQUIRED)`
Correct me if I'm wrong.
https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html

Reply:
i think you are. as the document puts,

> The find_dependency() macro wraps a find_package() call

if you are still confused, please check out its implementation. see https://github.com/Kitware/CMake/blob/master/Modules/CMakeFindDependencyMacro.cmake#L47

Reply:
But it was say "forwards the correct
  parameters for ``QUIET`` and ``REQUIRED`` which were passed to
  the original :command:`find_package` call"

https://github.com/Kitware/CMake/blob/ef5409100d310afd379e8f91d6ef429c741a8bad/Modules/CMakeFindDependencyMacro.cmake#L17-L19

This sets `REQUIRED` according to parent pkg:
https://github.com/Kitware/CMake/blob/ef5409100d310afd379e8f91d6ef429c741a8bad/Modules/CMakeFindDependencyMacro.cmake#L49

It using `find_package(JeMalloc REQUIRED)`, I think it will cause FATAL ERROR when calling `find_package(RocksDB QUIET)`.

Reply:
sorry, i misunderstood the use case. there is chance that caller does not "REQUIRE" rocksdb when trying to find it.

Reply:
I guess that's the reason why `find_dependency()` exists vs. `find_pacakge()`